fastavro/reader.py

```diff
@@ -491,6 +491,9 @@ def __iter__(self):
     def next(self):
         return next(self._records)
 
+    def __next__(self):
```

Comment:
I agree. updated.

On Mon, Aug 1, 2016 at 10:49 AM, Miki Tebeka <notifications@github.com>
wrote:

> In fastavro/reader.py
> <https://github.com/tebeka/fastavro/pull/59#discussion_r73001962>:
>
> > @@ -491,6 +491,9 @@ def __iter__(self):
> >      def next(self):
> >          return next(self._records)
> >
> > +    def __next__(self):
>
> I prefer
>
>     __next__ = next
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/tebeka/fastavro/pull/59/files/829e46a43571b5df833f9c8503dda44d68532f4a..996c1c241c25d6768da5b706c0d2bd5d570449de#r73001962>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHOTgBhQ0Bdb50Ip1d-lCUqUcK-mp2Tpks5qbhVwgaJpZM4JTJsY>
> .
>